File: Sharebook/Client/src/app/core/Core/Models/Properties/CategorySuggestion.ts
import {ICategorySuggestion} from "../../../Public/Models/Interfaces/ICategorySuggestion";

export class CategorySuggestion implements ICategorySuggestion {
  // [JsonProperty("name")]
  public name: string;

  // [JsonProperty("slug")]
  public slug: string;

  // [JsonProperty("size")]
  public size: number;
}

File: Sharebook/Server/src/Services/Post/Sharebook.Post.API/Models/CategorySuggestion.cs
namespace Sharebook.Post.API.Models
{
    using Newtonsoft.Json;

    using Sharebook.Common.Public.Models.Interfaces;

    public class CategorySuggestion : ICategorySuggestion
    {
        [JsonProperty("name")]
        public string Name { get; private set; }

        [JsonProperty("slug")]
        public string Slug { get; private set; }

        [JsonProperty("size")]
        public int Size { get; private set; }
    }
}
